src/Build/Logging/BinaryLogger/ProjectImportsCollector.cs

```diff
@@ -144,8 +144,8 @@ private void AddFileCore(string filePath)
                 return;
             }
 
-            using Stream entryStream = OpenArchiveEntry(filePath);
             using FileStream content = new FileStream(filePath, FileMode.Open, FileAccess.Read, FileShare.Read | FileShare.Delete);
+            using Stream entryStream = OpenArchiveEntry(filePath);
```

Comment:
Does this ordering matter?

Reply:
I'd revert it just to be safe and keep the change minimal, but at a glance seems benign.